### PR TITLE
Added FullPage functionality to screenshot feature

### PR DESCRIPTION
**Pull request by tantric-slug**
not sure about these conflicts

### CODE REVIEW

**Comment by tantric-slug** [comment]
> not sure about these conflicts

looks like I was using a different version of babel?

File: app.mjs [hunk 1]
```diff
@@ -56,8 +56,10 @@ import {promises as fs} from 'fs';
             page.setUserAgent(userAgent);
 
             const screenshotFile = `output/${ticketNumber}/${contents.screenshot.filename}`;
-            console.log(`Saving screenshot to ${screenshotFile}`);
+            const screenshotFileFull = `output/${ticketNumber}/${contents.screenshot.filename.slice(0,-4)}Full.png`;
```
Comment (williamtdr) [inline app.mjs]: Why are the last 4 characters being chopped off?

Comment (tantric-slug) [reply app.mjs]: I chopped of the last 4 (".png") because I wanted to add the "Full.png" suffix.

Comment (williamtdr) [reply app.mjs]: Ah, that makes sense. Thanks!

**Comment by rafibarash** [comment]
unblocking this cause this will fix an issue with running the honeyclient, but still think we should change the filtering and run for all requests in input

**Comment by williamtdr** [comment]
We should avoid re-processing completed jobs until we know how we’re getting the screenshot files out of the container. If they’re on a shared volume, we can’t delete them as they’ll be needed for page refreshes / links to history from the home page.

**Comment by rafibarash** [comment]
why can't we just override a previously processed job though? the filter is a pain for testing purposes and even in production if someone resubmits a page shouldn't the page be rechecked?

**Comment by williamtdr** [comment]
Because the jobs are enumerated by ID, if someone re-submits a page it’ll get a new ID. If they refresh a results page, it shouldn’t have to assemble all of the artifacts again - what if a site gets changed or taken down, etc. For testing, wipe the directory in-line: ‘rm -r output/*; npm start’

**Comment by rafibarash** [comment]
I see what you're saying. On that note, do we think ID is the best way to organize these? I think organizing by url would have an advantage over ID so we can clearly see what sites we have processed (at least while we are using file storage for these requests). This could also be easier for the frontend to read from

**Comment by williamtdr** [comment]
A good idea - I’ll see how hard it is to whip up based on primary key. It’s easy to have an auto incrementing field that gets filled in - so we could disambiguate based on instance of the snapshot + the site, like /run/umn.edu/3. I’ll see if postgres supports having auto incrementing as unique with another key.

**Pull request merged**